Docker build error.

Error message:

```
Step 8 : RUN make install

 ---> Running in 8694dfe97635

glide install

fatal: Not a git repository (or any of the parent directories): .git

make: glide: Command not found

Makefile:26: recipe for target 'install' failed

make: glide: Command not found

make: *** [install] Error 127

The command '/bin/sh -c make install' returned a non-zero code: 2
```

--- a/docker/Dockerfile.build
+++ b/docker/Dockerfile.build
@@ -1,9 +1,8 @@
-FROM golang:1.6.3-alpine
+FROM appleboy/golang-testing
 
 MAINTAINER Bo-Yi Wu <appleboy.tw@gmail.com>
 
 ENV VERSION #VERSION#
-RUN apk --update add git make curl
 RUN mkdir -p $GOPATH/src/github.com/appleboy/gorush
 Add build.tar.gz $GOPATH/src/github.com/appleboy/gorush/
 WORKDIR $GOPATH/src/github.com/appleboy/gorush

